scripts/receive_meteor.sh

```diff
@@ -112,51 +112,20 @@ if [ "$METEOR_RECEIVER" == "rtl_fm" ]; then
   log "Starting rtl_fm record" "INFO"
   ${AUDIO_PROC_DIR}/meteor_record_rtl_fm.sh $CAPTURE_TIME "${RAMFS_AUDIO_BASE}.wav" >> $NOAA_LOG 2>&1
 
+  sleep 2
```

Comment:
Why? Can we add a comment here explaining these otherwise some smartass like me will remove them.

(Also can't we test for something rather than just sleep? i.e. "if file doesn't exist sleep")

Reply:
either way. happy to see that improved - i think it just takes a second for the file to close..